tests/keras_tests/feature_networks_tests/feature_networks/weights_mixed_precision_tests.py

```diff
@@ -99,23 +99,21 @@ def compare(self, quantized_model, float_model, input_x=None, quantization_info=
 
 
 class MixedPercisionSearchTest(MixedPercisionBaseTest):
-    def __init__(self, unit_test, distance_metric=MpDistanceWeighting.AVG):
+    def __init__(self, unit_test, distance_metric=MpDistanceWeighting.AVG, expected_mp_config=[0,0]):
         super().__init__(unit_test, val_batch_size=2)
-
+        self.expected_mp_config = expected_mp_config
         self.distance_metric = distance_metric
 
     def get_resource_utilization(self):
-        # resource utilization is infinity -> should give best model - 8bits
-        return ResourceUtilization(np.inf)
+        return ResourceUtilization(17919)
 
     def get_mixed_precision_config(self):
         return mct.core.MixedPrecisionQuantizationConfig(num_of_images=1,
                                                          distance_weighting_method=self.distance_metric)
 
     def compare(self, quantized_model, float_model, input_x=None, quantization_info=None):
         conv_layers = get_layers_from_model_by_type(quantized_model, layers.Conv2D)
-        assert (quantization_info.mixed_precision_cfg == [0,
-                                                          0]).all()  # resource utilization is infinity -> should give best model - 8bits
+        assert (quantization_info.mixed_precision_cfg == self.expected_mp_config).all()
```

Comment:
I know it's not your original code, but if you already touched it, change the assert to some unitTest assertion